docs/specs/validation.yml

```diff
@@ -2,12 +2,6 @@
 # Check git merge conflict marker
 inputs:
   docfx.yml:
-  docs/toc.md: |
```

Comment:
I removed TOC.md check because if it contains merge conflict marker, it'll break TOC syntax and we'll throw anyway.